dwave/system/samplers/clique.py

```diff
@@ -103,11 +103,31 @@ def properties(self):
 
     @property
     def largest_clique_size(self):
-        """The maximum number of variables."""
+        """The maximum number of variables that can be embedded."""
         return len(self.largest_clique())
 
+    def clique(self, variables):
+        """Return the clique embedding of the given size.
+
+        Args:
+            variables (int/collection):
+                The source variables. If an integer, the variables are
+                embedding will be labelled `[0,n)`.
```

Comment:
Do you mean something like this?

                Source variables. If an integer, the variables 
                embedded are labelled `[0,n)`.